packages/wallets/src/features/cfd/screens/ChangePassword/TradingPlatformChangePasswordScreens.tsx

```diff
@@ -23,13 +27,12 @@ const MT5ChangePasswordScreens: React.FC<MT5ChangePasswordScreensProps> = ({ pla
     const { mutate } = useVerifyEmail();
     const { data: activeWallet } = useActiveWalletAccount();
 
+    const isMt5 = platform === PlatformDetails.mt5.platform;
```

Comment:
better to check the isDerivX to keep it consistent with the `packages/wallets/src/features/cfd/screens/ChangePassword/ChangePassword.tsx` file